components/sup/doc/render_context_schema.json

```diff
@@ -478,8 +478,17 @@
                 "properties": {
                     "first": {
                         "description": "The first member of this service group. If the group is running in a leader topology, this will also be the leader.",
+                        "$deprecated": "Since 0.56.0; if you want the leader, use `leader` explicitly. 'first' isn't deterministic, either, so you can just use `members[0]` instead",
```

Comment:
I'm not up on the whole handlebars world, but I remember seeing something about the syntax changing to something like `members.[0]`. Does that apply here?